explorer/models.py

```diff
@@ -98,34 +129,44 @@ def available_params(self):
         return p
 
     def get_absolute_url(self):
-        return reverse("query_detail", kwargs={'query_id': self.id})
+        return reverse("query_detail", kwargs={"query_id": self.id})
 
     def log(self, user=None):
-        if user and user.is_anonymous():
-            user = None
-        ql = QueryLog(sql=self.final_sql(), query_id=self.id, run_by_user=user)
+
+        if user:
+            # In Django<1.10, is_anonymous was a method.
+            if user.is_anonymous:
```

Comment:
@Ujjwal-Squadstack  These are the changes for that `bool` error previously `is_anonymous` was a method now it is property